### ISSUE TITLE
After DIM sync all tags are gone

### ISSUE THREAD

**Issue by smiller171** [DestinyItemManager/DIM#5010]
When enabling DIM sync, it brings over existing loadouts but not tags. I had to disable it, restore an old version of my Google Drive data, save a backup, enable DIM sync, then import the backup.

**Comment by bhollis** [comment]
Hmm, not sure exactly how that happened since loadouts and tags are stored in the same place. Can you open the DIM Sync log from settings and see what your first import said?

**Comment by philkernick** [comment]
I've just noticed that I have exactly the same problem.  Loadouts kept, all tags deleted.

**Comment by bhollis** [comment]
Can you explain exactly what you did to get in this situation? 

For example:

1. I started with X loadouts and Y tags that I had stored in Google Drive.
2. I loaded beta, and clicked "Enable Sync" in the notification.
3. I got a popup saying my data had been imported.
4. I could still see my loadouts, but all tags were gone.

**Comment by philkernick** [comment]
Ok.  Looking at my iPad which is still synced to Google Drive.

10 Destiny 2 loadouts
6 Destiny 2 items with tags or notes

On Chrome on Windows 10, I chose the enable DIM sync option.

It popped up a dialog box asking me to save dim-data.json.  There was no notification that it would do this, nor what I was supposed to do with it, so I just saved it.

Looking at the DIM Sync Change History, I see this:

`import | Imported 10 loadouts and 1 tagged items | dim-beta`

Now I see all my loadouts, but only 1 out of the 6 tagged items.
